lib/state/ui/middleware.ts

```diff
@@ -19,7 +19,7 @@ export const middleware: S.Middleware = store => {
       case 'App.deleteNoteForever':
       case 'App.notesLoaded':
       case 'App.restoreNote':
-      case 'App.selectTag':
+      case 'SELECT_TAG':
```

Comment:
Considering the semantics of what we're doing I would love to entertain drawing a clearer distinction between what we're doing here and what we do when we click on a tag inside the note editor.

For instance, could this be `OPEN_TAG` while clicking on or selecting the tag in the note editor would be `SELECT_TAG`? I doubt we'd make that a global Redux action but for the sake of our own confusion when discussing it maybe something like `OPEN_` could remove the ambiguity?

Reply:
Done.